pennylane/ops/op_math/controlled_ops.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains controlled operators based on the Controlled and ControlledOp class.
+"""
+
+import warnings
+from typing import Iterable
+
+from pennylane.operation import AnyWires
+from pennylane.ops.qubit.matrix_ops import QubitUnitary
+from .controlled_class import ControlledOp
+
+
+class ControlledQubitUnitary(ControlledOp):
+    r"""ControlledQubitUnitary(U, control_wires, wires, control_values)
+    Apply an arbitrary fixed unitary to ``wires`` with control from the ``control_wires``.
+
+    In addition to default ``Operation`` instance attributes, the following are
+    available for ``ControlledQubitUnitary``:
+
+    * ``control_wires``: wires that act as control for the operation
+    * ``control_values``: the state on which to apply the controlled operation (see below)
+    * ``target_wires``: the wires the unitary matrix will be applied to
+    * ``active_wires``: Wires modified by the operator. This is the control wires followed by the target wires.
+
+    **Details:**
+
+    * Number of wires: Any (the operation can act on any number of wires)
+    * Number of parameters: 1
+    * Number of dimensions per parameter: (2,)
+    * Gradient recipe: None
+
+    Args:
+        U (Union[array[complex], QubitUnitary]): square unitary matrix or a QubitUnitary operation
```

Comment:
It does if initialized using a `qml.QubitUnitary`. You can also pass it a matrix, which it will then use to create a `QubitUnitary` base. 

Passing a matrix `U` is the original behaviour, the option to directly pass it a base operator has been added in this PR to mimic the behaviour of SymbolicOp and CompositeOp, since I imagine this is what people would expect if they are used to those operators. But if people have been using it in the past, they would expect to pass just a matrix.

I'll expand a bit in the docstring to make this clearer.

Reply:
Thanks for expanding the docstring. But the name of the `arg` should be the same as the argument in the `__init__.py` method. Thus this should be `base`.

Reply:
Ah, thanks! I hadn't thought about that :)